sim/midas/src/main/scala/midas/passes/StroberUtils.scala

```diff
@@ -1,17 +1,19 @@
 // See LICENSE for license details.
 
-package strober
-package passes
+package midas.passes
 
 import firrtl._
 import firrtl.ir._
 import firrtl.Mappers._
 import firrtl.annotations._
-import core.ChainType
-import mdf.macrolib.SRAMMacro
-import mdf.macrolib.Utils.readMDFFromString
 import scala.collection.mutable.{ArrayBuffer, HashMap, HashSet, LinkedHashSet}
 
+/**
+  * This file contains legacy FIRRTL features that did not exist in upstream
+  * FIRRTL at the time of Strober's development. These continue to be used in:
```

Comment:
Seems like a reasonable compromise, moving to `midas.passes` but keeping the `Strober` name in `StroberUtils`.